bin/repo

```diff
@@ -176,6 +176,11 @@ module Repo
 		## cast to Array from single file
 		files = [files] if files.is_a?(String)
 		
+		if files.any? { |file| file.start_with?("type_data/") }
```

Comment:
@ondreian let me know if you think it makes more sense to do this outside of the `sync` method.